src/main/java/sirius/kernel/commons/Value.java

```diff
@@ -1943,9 +1946,12 @@ public Value translate() {
      * if the string representation of the wrapped value starts with {@code $}.
      * The dollar sign is skipped when passing the key to <tt>NLS</tt>. Otherwise <tt>this</tt> is returned.
      * @see NLS#get(String, String)
+     * @deprecated User {@link sirius.kernel.settings.Settings#getTranslatedString(String, String)} if this value is
+     * loaded from a Config/Settings. Otherwise use {@link NLS#smartGet(String, String)}
      */
     @Nonnull
     @CheckReturnValue
+    @Deprecated
```

Comment:
Replace usages within kernel otherwise we can't build without accepting deprecations.
`[WARNING] /drone/src/src/main/java/sirius/kernel/settings/Extension.java:[80,27] [deprecation] translate() in Value has been deprecated.`